js/npm/etherscan/package.json

```diff
@@ -21,13 +22,16 @@
     "Parity",
     "Promise"
   ],
-  "scripts": {
+  "peerDependencies": {
+    "es6-promise": "^4.0.5",
+    "isomorphic-fetch": "^2.2.1",
+    "babel-polyfill": "^6.22.0"
```

Comment:
Really wondering if https://github.com/es-shims are maybe not a more elegant (can include it without issues) solution. Worth a go.